fetch-config.sh

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
 
+region="eu-west-1"
+
 mkdir -p ~/.configuration-magic/
-aws s3 cp s3://guconf-flexible/atom-workshop/atom-workshop.conf ~/.configuration-magic/atom-workshop.conf --profile composer
+aws s3 cp s3://guconf-flexible/atom-workshop/atom-workshop.conf ~/.configuration-magic/atom-workshop.conf --profile composer --region $region
```

Comment:
strange that `s3 cp` needs a region

Reply:
Yes, it started demanding that after we encrypted the buckets.